### DISCUSSION TITLE
White list in dark mode - cannot read it

### DISCUSSION THREAD

**Discussion by Feriman22** [dani-garcia/vaultwarden#3559] [Q&A]
### Subject of the issue
Can't read the list in dark mode when edit rights of an user

### Deployment environment

* vaultwarden version: 2023.3.0
* Install method: Docker image
* Clients used: Chrome
* Reverse proxy and version: -
* MySQL/MariaDB or PostgreSQL version: -
* Other relevant details:

### Steps to reproduce
1., Go to Organizations --> Vault
2., Select any collection, press "three dots" on the right side, then "Edit info"
3., Go to "Access"
4., Select any user, and try to edit his permissions

### Expected behaviour
It should be in dark mode and readable

### Actual behaviour
Cannot read the list, because it's white

### Troubleshooting data
![image](https://github.com/dani-garcia/vaultwarden/assets/22872515/a77c47ea-55ee-4f75-8a83-ef4fa2f93c39)
![image](https://github.com/dani-garcia/vaultwarden/assets/22872515/b75af5df-dd8e-4c1c-9356-b4959274685e)

**Comment by BlackDex** [comment]
You probably have an extension installed like DarkReader and that is enabled for your vault domain.
If you disable this, it should probably be solved when you already have dark mode enabled for your web-interface.

Also, we can not help with this, since this is a client specific issues.
When this is still an issue for you, i suggest to test this on `vault.bitwarden.com` and report this upstream to Bitwarden.
